test/Engine/ProtoTest/Associative/Inheritance.cs

```diff
@@ -8,7 +8,7 @@ namespace ProtoTest.Associative
     class InheritaceTest : ProtoTestBase
     {
         [Test]
-        [Category("DSDefinedClass")]
+        [Category("DSDefinedClass_Ignored_DoNotTestInheritance")]

```

Comment:
Does this need a Category("Ignore") adding to it as well? or is this sufficient?